### ISSUE TITLE
More Pleasing Quest Rewards

### ISSUE THREAD

**Issue by Spyrox17** [Sunekaer/stoneBlock#230]
## Context
At the beginning of the mod, there was a few quest rewards that helped you kick-start your way into the game by giving you actual specific items. After that, everything is just random chance.

My suggestion is to give specific items throughout the majority of the quest book instead of leaving it to chance.

As an example, lets say I completed the dark matter quest. Instead of giving me a loot box with random shmandom stuff in it, it would be more ideal to give 1 dark matter as a reward.

It doesn't feel very rewarding when I complete something that took hours to complete and get no mid-game or even late-game rewards that are not even worth considering, from that quest (because it feels like the loot boxes only give you shit things 90% of the time, and when they don't give you shit things, they give you things you don't need until very-late-game).

TL;DR
Make the rewards ACTUAL rewards.

**Comment by Sunekaer** [comment]
Quest rewards won’t be changed

**Comment by neoc666** [comment]
Even cookies are better than viaducts or black lotuses, I don't know what devs are thinking when they make such lame 'rewards'
